### PR TITLE
Updates troubleshoot.md

### PR DESCRIPTION
**Pull request by gilzow**
removes mentions of the 5 minute execution hard cap in php troubleshooting section

### CODE REVIEW

**Comment by ralt** [comment]
The link seems to be wrong and should point here, no? https://docs.platform.sh/create-apps/app-reference/single-runtime-image.html#runtime

File: sites/platform/src/languages/php/troubleshoot.md [hunk 1]
```diff
@@ -35,8 +35,7 @@ WARNING: [pool web] child 120, script '/app/public/index.php' (request: "GET /in
 ```
 
 This means your PHP process is running longer than allowed.
-You can adjust the `max_execution_time` value in `php.ini`,
-but there is still a hard cap of 5 minutes on any web request.
+You can adjust the `max_execution_time` value in `php.ini`. See also `request_terminate_timeout`in the [App reference](/create-apps/app-reference.md#runtime).
```
Comment (gilzow) [inline sites/platform/src/languages/php/troubleshoot.md:38]:
```suggestion
You can adjust the `max_execution_time` value in `php.ini`. See also `request_terminate_timeout`in the [App reference](/create-apps/app-reference/single-runtime-image.md#runtime). 
```

**Review by gilzow — COMMENTED** [review]
update link since the src page changed between when I made the PR and now

File: sites/platform/src/languages/php/troubleshoot.md [hunk 2]
```diff
@@ -35,8 +35,7 @@ WARNING: [pool web] child 120, script '/app/public/index.php' (request: "GET /in
 ```
 
 This means your PHP process is running longer than allowed.
-You can adjust the `max_execution_time` value in `php.ini`,
-but there is still a hard cap of 5 minutes on any web request.
+You can adjust the `max_execution_time` value in `php.ini`. See also `request_terminate_timeout`in the [App reference](/create-apps/app-reference/single-runtime-image.md#runtime).
```
Comment (ralt) [inline sites/platform/src/languages/php/troubleshoot.md:38]: missing space before the 2nd `in`

Comment (gilzow) [inline sites/platform/src/languages/php/troubleshoot.md:38]:
```suggestion
You can adjust the `max_execution_time` value in `php.ini`. See also `request_terminate_timeout` in the [App reference](/create-apps/app-reference/single-runtime-image.md#runtime). 
```

**Review by AnouckColson — APPROVED** [review]

**Pull request merged**